### PR TITLE
Add GraphQL docs

### PR DESCRIPTION
**Pull request by tusharojha**
- New code block styles added
- Change the illustration of entities 
- Added GraphQL docs with examples

### CODE REVIEW

File: docs/develop/SubsocialGraphQL.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+id: subsocial-graqhql
+title: Subsocial GraqhQL
+displayed_sidebar: developSidebar
+---
+
+Building social apps need faster data fetching, and smoother experience for users.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:7]:
```suggestion
When you are building social apps, you need to fetch data quickly to provide a smooth experience for your users.
```

File: docs/develop/SubsocialGraphQL.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+Subsocial provides you the Subsocial's GraphQL endpoint, that can be used to query the blockchain instantaneously.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:9]:
```suggestion
Subsocial provides you with the Subsocial GraphQL endpoint, which can be used to query the blockchain instantaneously.
```

File: docs/develop/SubsocialGraphQL.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+### Getting Started
+
+You can add graphql and apollo client to a new or existing project.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:13]:
```suggestion
You can add GraphQL and the Apollo client to a new or existing project.
```

File: docs/develop/SubsocialGraphQL.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+To add create a new project, you can use [Subsocial Starter](/docs/develop/developer-quickstart).
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:15]:
```suggestion
To add create a new project, you can use the [Subsocial Starter](/docs/develop/developer-quickstart).
```

File: docs/develop/SubsocialGraphQL.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+      name
+      summary
+    }
+  }
+}
+`
+```
+
+The above query fetches the top 3 most liked posts of this year. 
+You can choose the data you need about the query and leverage the power of GraphQL.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:66]:
```suggestion
You can choose the data you need from the query, and leverage the power of GraphQL.
```

File: docs/develop/SubsocialGraphQL.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+      {
+        "content": "bafyreiacd3ae2maa3tbwu2eomsvysrfopowo6xmp4g2sprslfaapieayba",
+        "title": "December 2022 In Review",
+        "reactionsCount": 1,
+        "sharedPost": null,
+        "createdAtTime": "2023-01-03T15:13:36.362000Z",
+        "space": {
+          "id": "1",
+          "name": "Subsocial",
+          "summary": "Subsocial is an open protocol for decentralized social networks and marketplaces. It's built with Substrate and IPFS. Learn more"
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:115]: Alex just recently updated the space description, please update this

File: docs/develop/SubsocialGraphQL.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+      }
+    ]
+  }
+}
+```
+
+## More Examples
+
+### News Feed
+Get news feed of posts for an address.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:126]:
```suggestion
Get a news feed of posts for an address.
```
from an address? a feed of all the posts that an address has posted?
or is it building a news feed for a particular account based on what they follow?

File: docs/develop/SubsocialGraphQL.md [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+            }
+          }
+        }
+      }
+    }
+`
+```
+
+### Get Notifications
+Get notification data for a specific address, with aggregated true as filter to make the notification show only once.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:157]: "with aggregated true as filter"

I have no idea what you are trying to say here, so I can't even offer a suggestion. Please try to word it differently.

File: docs/develop/SubsocialGraphQL.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+        }
+      }
+    }
+`
+```
+
+### Get Notifications
+Get notification data for a specific address, with aggregated true as filter to make the notification show only once. 
+
+> The amount of same notifcations that are aggregated is in aggCount.
```
Comment (Yung-Beef) [inline docs/develop/SubsocialGraphQL.md:159]: "The amount of same notifcations"

Similar to the above comment

File: docs/develop/sdk/Installation.md [hunk 10]
```diff
@@ -6,13 +6,13 @@ displayed_sidebar: developSidebar
 
 To install the SDK, run:
 
-```
+```bash
 yarn add @subsocial/api
 ```
 
 Also, you can add additonal methods for utility using utils package. Install it like this:
```
Comment (Yung-Beef) [inline docs/develop/sdk/Installation.md:13]:
```suggestion
Also, you can add additonal methods for utility using the utils package. Install it like this:
```

File: docs/develop/sdk/Transactions.md [hunk 11]
```diff
@@ -16,7 +16,7 @@ So, these structures can be created or updated via [transactions](https://polkad
 
 You need to add 2 more dependencies for this
```
Comment (Yung-Beef) [inline docs/develop/sdk/Transactions.md:17]:
```suggestion
You need to add 2 more dependencies for this:
```

**Review by Yung-Beef — COMMENTED** [review]
Not always 100% sure on the context, so some of my suggestions may be wrong. Please review them

File: docs/develop/SubsocialGraphQL.md (hunk 7 above)
Comment (tusharojha) [reply docs/develop/SubsocialGraphQL.md:126]: for the address / user.

**Comment by siman** [comment]
@tusharojha Compress images with TinyPNG.com

File: docusaurus.config.js [hunk 12]
```diff
@@ -268,7 +269,7 @@ module.exports = {
 
         ],
         createRedirects(existingPath) {
-          console.log('existing path ',existingPath)
+          console.log('existing path ', existingPath)
```
Comment (olehmell) [inline docusaurus.config.js:272]: Is it really need in prod?

**Review by olehmell — APPROVED** [review]

**Pull request merged**